lib/fizzy/execute.cpp

```diff
@@ -534,10 +532,8 @@ inline constexpr T fmin(T a, T b) noexcept
 template <typename T>
 inline constexpr T fmax(T a, T b) noexcept
 {
-    if (std::isnan(a))
-        return a;
-    if (std::isnan(b))
-        return b;
+    if (std::isnan(a) || std::isnan(b))
+        return std::numeric_limits<T>::quiet_NaN();
```

Comment:
Where did this manifest itself as an error?

Reply:
In the NaN propagation test - for signaling NaN inputs, we must return an arithmetic NaN.